File: 119. Pascal's Triangle II/solution_3.java
class Solution {
    public List<Integer> getRow(int rowIndex) {
        List<Integer> res = new ArrayList<>();
        long num = 1;
        for (int i = 0; i <= rowIndex; i++) {
            res.add((int)num);
            num = num * (rowIndex - i) / (i + 1);
        }
        return res;
    }
}

File: 342. Power of Four/solution_3.cpp
class Solution {
public:
    bool isPowerOfFour(int num) {
        if (num <= 0)
            return false;
        return !(num & (num - 1)) && (num - 1) % 3 == 0;
    }
};
// !num&(num-1)表示2的指数幂